steps/hooks.go

```diff
@@ -31,6 +32,18 @@ var _ = gauge.BeforeScenario(func(exInfo *gauge_messages.ExecutionInfo) {
 	store["namespace"] = namespace
 	store["scenario.cleanup"] = cleanup
 	store["targetNamespace"] = config.TargetNamespace
+
+	// Skip pipelines SA check if scenario has @install tag
+	skipSAWait := false
```

Comment:
You don't need variable `skipSAwait` at all... just use if...else

Reply:
Done